lib/sentry/config.ex

```diff
@@ -0,0 +1,89 @@
+defmodule Sentry.Config do
+  @default_included_environments [:dev, :test, :prod]
+  @default_environment_name Mix.env
+  @default_max_hackney_connections 50
+  @default_hackney_timeout 5000
+  @default_exclude_patterns [~r"/_build/", ~r"/deps/", ~r"/priv/"]
+  @default_path_pattern "**/*.ex"
+  @default_context_lines 3
+  @default_sample_rate 1.0
+
+  def validate_config! do
+  end
+
+  def get_included_environments do
```

Comment:
I can't really see the module doing any updating, etc., so it should be safe to assume we're only ever getting.